spec/cc/engine/content_resolver_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+require "rubocop"
+require "cc/engine/content_resolver"
+
+module CC::Engine
+  describe ContentResolver do
+    cops = RuboCop::Cop::Cop.all
+    whitelist = File.read("./spec/support/docs_whitelist.txt").lines.map(&:chomp)
+
+    it "has cops" do
+      expect(cops.count).to eq 310
```

Comment:
I wondered if that was why you did it. I think if you wanted that sanity check, `> 0` would be sufficient. But since you're asking Rubocop directly for the cop classes, I also wouldn't expect this to randomly break & return 0: if Rubocop changes the interface, I would expect it to blow up much more severely than that.